conf/server.yaml.template

```diff
@@ -41,6 +41,8 @@ SERVER:
   # SSH password if not using a SSH key to connect
   SSH_PASSWORD:  # replace-with-ssh-passwd
   # Path to private ssh key to be used when connecting via SSH.
+  # Specify port number for ssh client, Default: 22
+  SSH_CLIENT.PORT:
```

Comment:
@yanpliu You could include it in the section below for ssh_client or maybe rename this to SSH_PORT (commented same on your MR)

Reply:
> @yanpliu Sorry, I missed this earlier, you could include it in the section below for ssh_client or maybe rename this to SSH_PORT (commented same on your MR)
@Gauravtalreja1 Thanks for your suggestion, I have already update the name to SSH_PORT

Reply:
@yanpliu Thanks for the update, but this doesn't seem to be matching again with your satqe-jenkins Mr, could you please make sure you commit the right updates?

Reply:
> @yanpliu Thanks for the update, but this doesn't seem to be matching again with your satqe-jenkins Mr, could you please make sure you commit the right updates?
@Gauravtalreja1 In the latest commit [Update server.yaml.template](https://github.com/SatelliteQE/robottelo/pull/9735/commits/88eaa82a81cdb8683b11a83a36a494cb41f10674) , I have already updated the name the description to "  # Specify port number for ssh client, Default: 22
  SSH_PORT:", the same as https://gitlab.cee.redhat.com/satelliteqe/satelliteqe-jenkins/-/merge_requests/672/diffs#2ca514a1887a699c4754fa1e378ad2fc01fdb428_56_57.  What else need I to do? please let me know.

Reply:
This isn't about descriptions, so there're two maps under server.yaml, `SERVER` and `SSH_CLIENT`, and what you're adding in MR#672  is into `SSH_CLIENT` section with setting name `SSH_PORT`, which can be called with the settings object of dynaconf as `settings.server.ssh_client.ssh_port`
And what you're adding here in this PR into `template` is under SERVER map with setting name `SSH_PORT`  which can be called with settings object of dynaconf as`settings.server.server.ssh_port`, that's different, and what you call `settings.server.ssh_client.port` in robottelo/ssh.py for get_client is different as well, which shouldn't be the case.

It should make sense now, but if not, please let me know, and I will be able to assist you

Reply:
> This isn't about descriptions, so there're two maps under server.yaml, `SERVER` and `SSH_CLIENT`, and what you're adding in MR#672 is into `SSH_CLIENT` section with setting name `SSH_PORT`, which can be called with the settings object of dynaconf as `settings.server.ssh_client.ssh_port` And what you're adding here in this PR into `template` is under SERVER map with setting name `SSH_PORT` which can be called with settings object of dynaconf as`settings.server.server.ssh_port`, that's different, and what you call `settings.server.ssh_client.port` in robottelo/ssh.py for get_client is different as well, which shouldn't be the case.
> 
> It should make sense now, but if not, please let me know, and I will be able to assist you

@Gauravtalreja1 Have already port the ssh_port to `SSH_CLIENT`